docs/OIDC_provider_setup.md

```diff
@@ -0,0 +1,143 @@
+![image](https://user-images.githubusercontent.com/93269128/200741383-78a50796-bbee-4788-be9f-e608b7c1a95c.png)
+9. Click the “Assign” same line with the person to assign the application to it. 
+![image](https://user-images.githubusercontent.com/93269128/200741506-8dfdb745-af04-4f5f-a921-5429d4f60acd.png)
+10. Click the “Assign” same line as group to assign group to it. If assigned by group, all people in the group can login to this application. 
+![image](https://user-images.githubusercontent.com/93269128/200741698-a50cfb3c-d95d-45fc-bc0d-6784eda1d694.png)
+
+### OKTA PEOPLE SETUP
+
+1. Create new people.
+![image](https://user-images.githubusercontent.com/93269128/200742866-343db4c5-c0be-4b18-bcc1-a7e884be0512.png)
+2. Fill in the relative field.If the Activation is set to "Activate later", users will receive an email to set their password and activate the account.
+![image](https://user-images.githubusercontent.com/93269128/200743304-a3dee19b-5991-44a1-a44f-c0449005bdf8.png)
+3. If the Activation is set to "Activate now", it will show some options for setting the password. If the "I will set password" option is ticked, you can enter a temporary password and can specify user to change their password after first login or not.
+![image](https://user-images.githubusercontent.com/93269128/200754546-3ab07d68-f246-4f61-b192-835c84b529a8.png)
+
+### OKTA GROUP SETUP
+
+1. Create a new Group.  
+![image](https://user-images.githubusercontent.com/93269128/200758216-cce03795-e0d3-44fa-83fa-43fa2ec5702b.png)
+2. Give a name to the group and add description if you want to.  
+![image](https://user-images.githubusercontent.com/93269128/200758516-6c12f22c-6b08-4ca7-95f3-3dd8fb94cf12.png)
+3. Click the group name you create to manage the group.  
+![image](https://user-images.githubusercontent.com/93269128/200758916-dbf62e54-cc4e-4b53-8a1a-a9f25799caeb.png)
+4. Click the "Assign people" to add person to the group by clicking the person. You also can remove user from group by click "remove".
+![image](https://user-images.githubusercontent.com/93269128/200759364-4079f96a-b03c-4805-a18c-a7613384f0de.png)
+![image](https://user-images.githubusercontent.com/93269128/200759757-f29d297a-a514-40cf-b964-f4a9712bb05b.png)
+
+
+## AUTH0 SETUP
+
+### AUTH0 OIDC SETUP
+
+1. After signing an account for Auth0, select “Company” as the account type so you can limit the person who can log in to this organization. Fill in the company name and select size for company.	
+![image](https://user-images.githubusercontent.com/93269128/159425323-b0b2d57c-bfed-4887-aebd-7a4e54919d20.png)
+2. You can change the domain name and choose the country you want. Change of country will change the host url. 
+![image](https://user-images.githubusercontent.com/93269128/159425368-8f7b0fa1-9106-4c51-a4c6-eed5e7586da2.png)
+3. Click the “Application” in the Application section to go to the application page. Click “Create Application” to create a new application. 
```

Comment:
Under the application panel, click “Create Application” to create a new application ***